Analysis/TrueJet/examples/truejet_CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+########################################################
```

Comment:
I assume this is the `CMakeLists.txt` file that allows a standalone compilation of the `TrueJet` module? In that case, we could move this up to `Analysis/TrueJet`. It doesn't interfere there with the normal build process for MarlinReco, and it would make building TrueJet in a standalone fashion a bit easier, since there is no renaming of the `trujet_CMakeLists.txt` file necessary.